server/portal/settings/settings_custom.example.py

```diff
@@ -237,5 +237,6 @@
     "extractApp": 'extract',
     "makePublic": False,
     "hideApps": False,
-    "hideDataFiles": False
+    "hideDataFiles": False,
+    "onboardingSetupCompleteNext": '/workbench/'
```

Comment:
What do you think about something like `onboardingCompleteRedirect`?

Reply:
I like it; I'll change it 🎗️